docs/operations/scheduler.md

```diff
@@ -0,0 +1,111 @@
+
+## Creating a Custom Scheduler
+KubeVirt is compatible with custom schedulers. The configuration steps are described in the [Official Kubernetes 
+Documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers).
+Nevertheless, the Kubernetes version KubeVirt is running on and the Kubernetes version used to build the custom
+scheduler have to match.
+To get the Kubernetes version KubeVirt is running on, you can run the following command:
+
+```shell
+$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.13", GitCommit:"a43c0904d0de10f92aa3956c74489c45e6453d6e", GitTreeState:"clean", BuildDate:"2022-08-17T18:28:56Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
+Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.13", GitCommit:"a43c0904d0de10f92aa3956c74489c45e6453d6e", GitTreeState:"clean", BuildDate:"2022-08-17T18:23:45Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
+```
+
+Pay attention to the `Server` line. 
+In this case, the Kubernetes version is `v1.22.13`.
+You have to checkout the matching Kubernetes version and [build the Kubernetes project](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#package-the-scheduler):
+
+```shell
+$ cd kubernetes
+$ git checkout v1.22.13
+$ make
+```
+
+Then, you can follow the configuration steps described [here](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/).
+Additionally, the ClusterRole `system:kube-scheduler` needs permissions to use the verbs `watch`, `list` and `get` on StorageClasses.
+
+```yaml
+- apiGroups:                                                                                                   
+  - storage.k8s.io                                                                                             
+  resources:                                                                                                   
+  - storageclasses                                                                                             
+  verbs:                                                                                                       
+  - watch                                                                                                      
+  - list                                                                                                       
+  - get 
+```
+
+
+## Scheduling VMs with the Custom Scheduler
+
+The second scheduler should be up and running. You can check it with:
+
+```shell
+$ kubectl get all -n kube-system
+```
+
+The deployment `my-scheduler` should be up and running if everything is setup properly.
+In order to launch the VM using the custom scheduler, you need to set the `SchedulerName` in the VM's spec to `my-scheduler`.
+Here is an VM definition example:
```

Comment:
```suggestion
Here is an example VM definition:
```

Reply:
Done. I also have added an example use case. Please, check if you consider it appropriated. Thanks for your review :)